meson.build

```diff
@@ -0,0 +1,208 @@
+conf.set('HAVE_OLD_UPOWER', upower_old.found())
+
+
+if get_option('with-gconf')
+  gconf     = dependency('gconf-2.0')
+else
+  gconf     = dependency('', required: false)
+endif
+conf.set('HAVE_GCONF', gconf.found())
+
+
+gio_unix    = dependency('gio-unix-2.0',      required: false)
+libelogind  = dependency('libelogind',        required: false)
+libsystemdl = dependency('libsystemd-login',  required: false)
+if not libsystemdl.found()
+  libsystemdl=dependency('libsystemd',        required: false)
+endif
+
+if gio_unix.found() and libsystemdl.found()
+  logind = declare_dependency(dependencies: [ gio_unix, libsystemdl ])
+else
+  logind = dependency('', required: false)
+endif
+conf.set('HAVE_LOGIND', logind.found())
+
+if gio_unix.found() and libelogind.found()
+  elogind = declare_dependency(dependencies: [ gio_unix, libelogind ])
+else
+  elogind = dependency('', required: false)
+endif
+conf.set('HAVE_ELOGIND', elogind.found())
+
+conf.set10('HAVE_QT57', get_option('with-qt5-theme-support'))
+
+
+xtest   = dependency('xtst',    required: false)
+conf.set('HAVE_XTEST', xtest.found())
+
+xrender = dependency('xrender', required: false)
+conf.set('HAVE_XRENDER', xrender.found())
+
+docbook_enabled = get_option('with-docbook')
+if docbook_enabled
+  xmlto = find_program('xmlto')
+endif
+conf.set('DOCBOOK_DOCS_ENABLED', docbook_enabled)
+
+
+# Execinfo support
+conf.set10('HAVE_EXECINFO_H', cc.has_header('execinfo.h'))
+backtrace = cc.find_library('backtrace',  required: false)
+execinfo  = cc.find_library('execinfo',   required: false)
+
+
+# Check for X transport interface - allows to disable ICE Transports
+# See also https://bugzilla.gnome.org/show_bug.cgi?id=725100
+if get_option('with-xtrans')
+  xtrans = dependency('xtrans')
+else
+  xtrans = dependency('', required: false)
+endif
+conf.set10('HAVE_XTRANS', xtrans.found())
+
+# Check whether IPv6 is enabled on the system...
+have_ipv6 = false
+ipv6_libs = []
+if get_option('with-ipv6')
+  if cc.compiles('''
+    #include <sys/types.h>
+    #include <sys/socket.h>
+    int main() {
+      socket(AF_INET6, SOCK_STREAM, 0);
+      return 0;
+    }
+    ''')
+    if cc.has_function('getaddrinfo')
+      have_ipv6 = true
+    else
+      ipv6_test_libraries = [
+        cc.find_library('bsd', required: false),
+        cc.find_library('socket', required: false),
+        cc.find_library('inet', required: false),
+      ]
+      foreach lib: ipv6_test_libraries
+        if (not have_ipv6) and cc.has_function('getaddrinfo', dependencies: lib)
+          have_ipv6 = true
+          ipv6_libs = lib
+        endif
+      endforeach
+    endif
+  endif
+endif
+conf.set('ENABLE_IPV6', have_ipv6)
+
+################################################################################
+# Additionnal flags
+
+rootInclude = include_directories('.')
+
+pkgdatadir = join_paths(get_option('datadir'), meson.project_name())
+libexecdir = join_paths(get_option('libdir'),  meson.project_name())
```

Comment:
Please change ('libdir') to ('libexecdir')

Reply:
Libexecdir does not exist... That's why I'm creating this variable.

Reply:
You set it at build time

```
meson --libexecdir=/usr/libexec  whatever
```

https://github.com/mesonbuild/meson/commit/11f63105d003e222576a4e23ab106af2fc5a7741

Reply:
You will also need to add --libexecdir=whatever to debian rules file to suit.

Reply:
Perhaps you should look at

https://github.com/linuxmint/nemo/commit/eb342fd1569ad95b48bd6e2fa0555f1d1726fdc9
https://github.com/linuxmint/nemo/commit/c84980ab8583dfef8599c5c6aa5889e9fb3e78ae

Reply:
That's a feature coming in Meson 0.45.